doc/reference/storage/flash_map/flash_mpa.rst

```diff
@@ -0,0 +1,36 @@
+.. _flash_map_flash_area:
+
+Flash map (NVS)
+##########################
+
+Flash map is a way for storing flash partitioning information in one central
+distribution location in flash_area structures array form.
+
+Flash map is generated from DTS based on content of :ref:`flash_partitions`
+nodes.
+I order to use data contain in flash_map the flash_area API is provided.
+flash_area_open() API is the interface for obtaining given flash partitions
+flash_area from the flash map.
+
+
+Flash Area API (flash_area)
+###########################
+
+Flash_Area conception combines method for operation on flash chunk
+with description of this chunk. Its methods are basically wrappers around
+flash API with addition of input parameters range check and so. Not all flash
+operation are wrapped so API for retrieve flash area driver is included as well.
```

Comment:
```suggestion
operation are wrapped, so the API for retrieving data using the flash area driver is included as well.
```

Reply:
API for retrieve flash area driver - it really mean: to get the appropriate flash driver name which makes possible to operate on the area "by hand" using directly the flash driver API.

Reply:
@dbkinder ^^